src/azure-cli/azure/cli/command_modules/storage/_client_factory.py

```diff
@@ -196,16 +197,19 @@ def get_account_url(cli_ctx, account_name, service):
 
 def cf_blob_service(cli_ctx, kwargs):
     from knack.util import CLIError
+    client_kwargs = {}
     t_blob_service = get_sdk(cli_ctx, ResourceType.DATA_STORAGE_BLOB,
                              '_blob_service_client#BlobServiceClient')
     connection_string = kwargs.pop('connection_string', None)
     account_name = kwargs.pop('account_name', None)
     account_key = kwargs.pop('account_key', None)
     token_credential = kwargs.pop('token_credential', None)
     sas_token = kwargs.pop('sas_token', None)
+
+    client_kwargs.update(_prepare_client_kwargs_track2(cli_ctx))
```

Comment:
We can simply use `client_kwargs = _prepare_client_kwargs_track2(cli_ctx)` if there is no modification before this line.

Reply:
good catch. updated